app/.env

```
@@ -0,0 +1 @@
+BO_URL=http://localhost:1337/
```

Should be implemented inside .env.local
if the value doesnt exist here, he will search in other .env corresponding of environement (normally)
```suggestion 
BO_URL=
```

---

guess you right i need to check this up